### ISSUE TITLE
Identifier not found: [Length]

### ISSUE THREAD

**Issue by joshjdevl** [FStarLang/FStar#167]
Running the one time pad example from the paper

```
module OneTimePad
open Bytes
open String
open List
open Array

type block=b:bytes{Length b=n}
```

Error is
```
Identifier not found: [Length] 
```

**Comment by catalin-hritcu** [comment]
Where are you taking these examples from? They seem F* 0.7, not F* 1.0.

From README.md: "This is a new variant of F* that is still in development and we hope will lead to a 1.0 release soon. This new variant is incompatible and quite different compared to the previously released 0.7 versions."

**Comment by joshjdevl** [comment]
@catalin-hritcu [POPL 2014 paper](http://research.microsoft.com/apps/pubs/default.aspx?id=204802) which states in Table 1 that the examples compile and were tested and verified.
